core/client/app/templates/post-settings-menu.hbs

```diff
@@ -14,7 +14,7 @@
                 <span class="input-icon icon-link">
                     {{gh-input class="post-setting-slug" id="url" value=slugValue name="post-setting-slug" focus-out="updateSlug" selectOnClick="true" stopEnterKeyDownPropagation="true"}}
                 </span>
-                {{gh-url-preview slug=slugValue tagName="p" classNames="description"}}
+                <a class="description" target="{{if model.isDraft '_blank'}}" href="{{if model.isDraft model.previewUrl model.absoluteUrl}}">{{model.absoluteUrl}}</a>
```

Comment:
Uh, this breaks what the url preview does and is stupid. Bad code. Now instead of a "you will be this url" it does "you are this url"